### ISSUE TITLE
If Addin on webserver is unavailable, Outlook addin on browser timesout after 5 min but on native app, it timesout after 15 secs

### ISSUE THREAD

**Issue by mkondadg** [OfficeDev/office-js#3946]
# Provide required information needed to triage your issue
If Addin on webserver is unavailable, Outlook addin on browser timesout after 5 min but on native app, it timesout after 15 secs

## Your Environment
Windows desktop
Outlook
You have Microsoft Outlook Version 1.2023.1018.300 (Production).
Client Version is 20231027003.07 .
Windows 10 Enterprise 21H2 19044.2006
 
## Expected behavior
Addin timeout should be consistent on different browsers (chrome, firefox, edge, safari on mac) and native outlook app (windows, mac)

## Current behavior
Outlook addin on browser timesout after 5 min but on native app, it timesout after 15 secs

## Steps to reproduce
Deploy addin and make the webserver (where addin is hosted) down or
Deploy addin where webserver URL is invalid in manifest.xml

1. Send mail using native app and addin is triggered but timesout after 15 secs.
2. Send mail using firefox and addin is triggered but timesout after 5 mins
3. Send mail shows smart alert within 5 secs if addin doesnt respond quickly.

## Context
User experience is not consistent when addin is not available and user has to wait for very long time (5 min) to see the error

**Comment by Adrian-MSFT** [comment]
Hi @mkondadg, just to clarify, the timeout differences are happening when sending an email with an on-send add-in installed, is that correct?

**Comment by mkondadg** [comment]
yes @Adrian-MSFT , the timeout differences are happening when sending an email with an on-send add-in installed

**Comment by JuaneloJuanelo** [comment]
@mkondadg  can you please clarify in what platform is this issue? is this Outlook for the Web, new Outlook? Outlook for windows? all?

**Comment by mkondadg** [comment]
@JuaneloJuanelo this is for new outlook on windows.
We tested addin deployed using native app and browser (both using same user account on which addin is deployed)

**Comment by JuaneloJuanelo** [comment]
jsut for completeness. on  traditional outlook for Windows this should fail immediatly, right?

**Comment by Adrian-MSFT** [comment]
In addition to the last question, are you using the [ItemSend (on-send V1)](https://learn.microsoft.com/en-us/office/dev/add-ins/outlook/outlook-on-send-addins?tabs=windows) feature or the [Smart Alerts (on-send V2)](https://learn.microsoft.com/en-us/office/dev/add-ins/outlook/onmessagesend-onappointmentsend-events) feature?

**Comment by mkondadg** [comment]
@JuaneloJuanelo On traditional outlook, addin's will be disabled if COM sensor is already active. Below entry will disable new outlook addin and hence it will not fail in this case
```

      <EquivalentAddins>
        <EquivalentAddin>
          <ProgId>prog-id-company</ProgId>
          <Type>COM</Type>
        </EquivalentAddin>
      </EquivalentAddins>

```

**Comment by mkondadg** [comment]
@Adrian-MSFT we are using [Smart Alerts (on-send V2)]

**Comment by exextoc** [comment]
Thanks for reaching out. This bug in our backlog but unfortunately we donot have nay timelines to share at this point.
Internal tracking id- 246718

**Comment by mkondadg** [comment]
We also observed that "Smart Alert" is generated within 5 secs if addin doesnt respond quickly.
